docs/docs-components/IllustrationSection.js

```diff
@@ -3,7 +3,8 @@ import { Box, Flex, Heading } from 'gestalt';
 import { type Node } from 'react';
 import IllustrationContainer from './IllustrationContainer.js';
 
-const MIN_SVG_ILLUSTRATION_WIDTH = 285;
+// Matches minWidth in IllustrationCard
+const MIN_SVG_ILLUSTRATION_WIDTH = 245;
```

Comment:
Can we use an actual shared const here instead of relying on manually keeping them in sync?